test/ibmq/test_account_client.py

```diff
@@ -248,6 +250,42 @@ def test_job_cancel(self):
                     # just retry.
                     self.assertIn('JOB_NOT_CANCELLED', str(ex))
 
+    def test_access_token_not_in_exception_chain(self):
+        """Check that access token is replaced within chained request exceptions."""
+        backend = self.provider.backends.ibmq_qasm_simulator
+        circuit = transpile(self.qc1, backend, seed_transpiler=self.seed)
+        qobj = assemble(circuit, backend, shots=1)
+
+        api = backend._api
+        job = backend.run(qobj)
+        job_id = job.job_id()
+
+        session_ = api.client_api.session
+        access_token_ = session_.access_token
+
+        exception_message = 'The access token in this exception ' \
+                            'message should be replaced: {}'.format(access_token_)
+        self.assertIn(access_token_, exception_message)
```

Comment:
Is this assert really necessary? You did just format the message with the token right above...